ql/src/java/org/apache/hadoop/hive/ql/optimizer/calcite/HiveRelBuilder.java

```diff
@@ -158,7 +158,6 @@ public static SqlAggFunction getRollup(SqlAggFunction aggregation) {
     return null;
   }
 
-  @Override
```

Comment:
investigate this

Reply:
Didn't find any usage in `HiveRelBuilder.java` or `RelBuilder.java` in 1.21

Reply:
CALCITE-3774 have removed this method - and replaced it with `config.bloat()` 

as the comment suggests this feature caused incorrectly merged `Project`-s which contained `Over` expressions
* the new code have this bloat enabled by default with 100
* since the new check seems is heuristic based - it may not happen by default

I'm not sure if our issue is gone now or not - setting bloat to `-1` might be a way to disable this; and consider enabling it later